draft-ietf-quic-transport.md

```diff
@@ -2918,7 +2918,9 @@ received packets in preference to packets received in the past.
 
 An endpoint SHOULD treat receipt of an acknowledgment for a packet it did not
 send as a connection error of type PROTOCOL_VIOLATION, if it is able to detect
```

Comment:
s/for a packet it did not send/for a packet that the server would not have processed/

The change is necessary because 0-RTT packets are sent (but not expected to be processed).

Reply:
This applies in both directions though. Maybe we shouldn't update the transport doc at all?

Reply:
My bad, I think the replacement should be "for a packet that the peer would not have processed".